Docker/python-nginx/python2.7-alpine3.7/uwsgi.conf

```diff
@@ -0,0 +1,35 @@
+server {
+    listen 80;
+
+    location / {
+        uwsgi_param REMOTE_ADDR $http_x_forwarded_for if_not_empty;
+        uwsgi_param REMOTE_USER $http_x_userid if_not_empty;
+        uwsgi_param REMOTE_REQID $http_x_reqid if_not_empty;
+        uwsgi_param REMOTE_SESSIONID $http_x_sessionid if_not_empty;
+        uwsgi_param REMOTE_VISITORID $http_x_visitorid if_not_empty;
+
+        include uwsgi_params;
+        uwsgi_pass unix:/var/run/gen3/uwsgi.sock;
+        uwsgi_read_timeout 45s;
```

Comment:
Make READ_TIMEOUT configurable somehow via environment variable or whatever - just have `dockerrun` run sed or something - so we can give slow services long timeouts if we want to ...